flask_rebar/rebar.py

```diff
@@ -16,6 +16,8 @@
 from copy import copy
 from distutils.version import LooseVersion
 from functools import wraps
+from inspect import isclass
```

Comment:
```suggestion
```
(Would be unused after my suggestion below.)

Reply:
Per my response on the other convo, we do still need to do the extra check (to detect when we are/aren't passed a dictionary) so leaving this one in for now, but I'm also open to suggestions for a better way to accomplish that check without breaking any backwards compatibility so leaving this one here for now.